astropy/modeling/tests/test_models.py

```diff
@@ -768,3 +769,16 @@ def test_parameter_inheritance():
     assert b.h == 5
     assert b.f ==3
     assert b.f.fixed == True
+
+
+class ExponentialAndLogarithmicTester:
```

Comment:
There might be a better way to test this, but I'll defer to the sub-package maintainer.